tools.go

```diff
@@ -1,8 +1,9 @@
+//go:build tools
```

Comment:
Automatically added by go fmt in 1.17. It's not needed, and it's being added so that comments can be backwards compatible or something... https://golang.org/doc/go1.17#build-lines